tests/engine/interaction/pointer/parabola/ignore/test.md

```diff
@@ -0,0 +1,21 @@
+# Parabola ignore test
+## Run this script URL: [Manual](./test.js?raw=true)   [Auto](./testAuto.js?raw=true)(from menu/Edit/Open and Run scripts from URL...).
+
+## Preconditions
+- In an empty region of a domain with editing rights.
+
+## Steps
+Press 'n' key to advance step by step
+
+### Step 1
+- Move back to see all the objects
+### Step 2
+- Initial position
+- ![](./ExpectedImage_00000.png)
+### Step 3
+- Position 
+### Step 4
+- Position 
+- ![](./ExpectedImage_00001.png)
+### Step 5
+- Clean up
```

Comment:
Hm this file didn't seem to generate properly.

Reply:
Updated.